include/swift/AST/Expr.h

```diff
@@ -4916,7 +4916,10 @@ class KeyPathExpr : public Expr {
     const ProtocolConformanceRef *SubscriptHashableConformancesData;
     
     union {
-      unsigned SubscriptSize;
+      struct {
+        unsigned subscripConformancestSize;
```

Comment:
Typo - `subscriptConformancesSize`

Reply:
I'd suggest we rename this to `numLabels`, and `numConformances` which is more concise and `SubscriptSize` to `SubscriptInfo`